mailet/standard/src/test/java/org/apache/james/transport/matchers/HostIsLocalTest.java

```diff
@@ -46,7 +46,7 @@
 
     @Before
     public void setUp() throws Exception {
-        mockedMail = new FakeMail();
+        mockedMail = FakeMail.builder().build();
```

Comment:
I've tried the same thing, and it's not working as some mailets are changing some attributes of the Mail.
We really have to rething the Mail interface usage after v3.0 release.